CorsixTH/Lua/hospital.lua

```diff
@@ -2502,3 +2502,7 @@ end
 --! The UI parts of earthquake ticks
 function Hospital:tickEarthquake(stage)
 end
+
+--! Give advice that a patient is waiting for the player to build a GP's office
+function Hospital:adviseNoGpsOffice()
```

Comment:
```suggestion
function Hospital:adviseNoGpOffice()
```
Please drop the `s`. The room name is `general practioner office` afaik.

Reply:
I would also rather see GP formatted in all caps but that's quite the nit 🙃